eng/ReferenceAssemblies.props

```diff
@@ -11,7 +11,13 @@
     <IntermediateOutputPath>$(ArtifactsObjDir)ref/$(MSBuildProjectName)/$(Configuration)</IntermediateOutputPath>
 
     <!-- disable warnings about unused fields -->
-    <NoWarn>$(NoWarn);0169;0649</NoWarn>
+    <NoWarn>$(NoWarn);0169;0649;CS8618</NoWarn>
```

Comment:
I believe the `CSnnnn` and the `nnnn` formats are interchangeable, we could try to pick one.. Maybe `CSnnnn` is slightly better because other tools might error if we accidentally feed them `CSnnnn` ? I don't care though and no need to fix it here.

Reply:
I'll fix it here